src/renderers/shaders/ShaderLib/depth_vert.glsl

```diff
@@ -12,10 +12,18 @@ void main() {
 
 	#include <skinbase_vertex>
 
+	#ifdef USE_DISPLACEMENTMAP
```

Comment:
Normals are only used for displacement. Prevent unnecessary computation of normals if displacement is not used. Is this okay or is this decision needlessly complex?